strictdoc/core/file_tree.py

```diff
@@ -194,9 +194,7 @@ def find_files_with_extensions(
                 if (
                     not d.startswith(".")
                     and (d == "_static" or not d.startswith("_"))
-                    and d != "build"
-                    and d != "output"
-                    and d != "Output"
+                    and d not in ("build", "output", "Output")
```

Comment:
Let's separate the trivial linting fixes from the actual meat of this PR. I opened a separate PR to first sort out the new Ruff checks: https://github.com/strictdoc-project/strictdoc/pull/1922.

Reply:
Thanks as well